components/Header/index.js

```diff
@@ -9,4 +9,25 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+function header() {
+  let header = document.createElement("div");
+  let date = document.createElement("span");
+  let title = document.createElement("h1");
+  let temp = document.createElement("span");
```

Comment:
Remember to use `const` unless there is a reason you would need to change the variable.